core/src/mindustry/ui/dialogs/ModsDialog.java

```diff
@@ -133,11 +144,121 @@ void setup(){
+                                                                                }
+                                                                            });
+                                                                        } catch (Throwable e) {
+                                                                            modError(e);
+                                                                        }
+                                                                    }
+                                                                }, t2 -> Core.app.post(() -> modError(t2)));
+                                                            }, t2 -> Core.app.post(() -> modError(t2)));
+                                                        };
+                                                        installMod[0].run();
+                                                    });
+                                                    modSelected.buttons.button("$mods.github.open", Icon.github, () -> {
+                                                        openGithubPage[0] = () -> {
+                                                            Core.app.openURI("https://github.com/" + modsbrolist.repo);
+                                                        };
+                                                        openGithubPage[0].run();
+                                                    });
+                                                    modSelected.buttons.button("$back", Icon.left, () -> {
+                                                        modSelected.clear();
+                                                        modSelected.hide();
+                                                    });
+                                                    modSelected.keyDown(KeyCode.escape, modSelected::hide);
+                                                    modSelected.keyDown(KeyCode.back, modSelected::hide);
+                                                    modSelected.show();
+                                                }).width(480f).growX().left().fillY();
+                                                tablebrow.row();
+                                            }
+                                        }
+                                    }, error -> {
+                                        ui.showErrorMessage(error.toString());
+                                    });
+                                };
+                                rebuildBrowser[0].run();
+                            });
+                            dialog2.addCloseButton();
+                            dialog2.show();
+                            //Keep this until mods work
+                            ui.showErrorMessage("This feature doesnt work completely due to mods not working in v6. Wait for this to be fixed. If you decide to use it anyways, mods will not let you enter the game.");
```

Comment:
This should not be necessary. Incompatible mods (e.g. with min-game-version < 105) should not be installable at all.

Reply:
Most mods dont work at all in v6, but work in v104.6, so u added the warning. If you think it's unnecessary ill delete it

Reply:
I'm saying the mod browser should display incompatible mods instead of a startup dialog. You may need to wait until I make incompatibility more explicit for mods.

Reply:
Okay, then i delete the message and you'll do whatever you need to do